### PR TITLE
feat: add update / delete to faiss indexer

### PR DESCRIPTION
**Pull request by cristianmtr**
This requires https://github.com/jina-ai/jina/pull/1415

### CODE REVIEW

File: indexers/vector/FaissIndexer/tests/test_faissindexer.py [hunk 1]
```diff
@@ -77,6 +77,32 @@ def test_faiss_indexer_known(metas):
         assert idx.shape == (4, 2)
         np.testing.assert_equal(indexer.query_by_id([7, 4]), vectors[[3, 0]])
 
+    # update
```
Comment (JoanFM) [inline indexers/vector/FaissIndexer/tests/test_faissindexer.py:80]: Can you duplicate the test adding the update feature? to have it isolated?

Comment (cristianmtr) [reply indexers/vector/FaissIndexer/tests/test_faissindexer.py:112]: Done

**Review by maximilianwerk — APPROVED** [review]
LGTM

**Comment by JoanFM** [comment]
I think we need to have a release with the changes in the core already there

**Review by florian-hoenicke — APPROVED** [review]

**Pull request merged**